.github/ISSUE_TEMPLATE/eia_new_year.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Integrate New Year of EIA data
+about: Check-list for integrating a new year of EIA data
+title: ''
+labels: new-data
+assignees: ''
+
+---
+
+### EIA new year of data integration check-list:
+
+Review Annual Updates Docs: https://catalystcoop-pudl.readthedocs.io/en/dev/dev/annual_updates.html
+
+Replace references to `EIA_DATASET_NAME` with either `eia_860`, `eia923`, or `eia861` 
+depending on which EIA dataset you are updating.
+
+- [ ] Add the new Zenodo archive DOI to `pudl/workspace/datastore.py`.
+- [ ] Run the datastore script to download the new raw data: `pudl_datastore --dataset 
+EIA_DATASET_NAME`. The new raw data will appear in `pudl_input/eia923/<ZENODO_DOI>/...`
+- [ ] Update the mapping information in `pudl/package_data/EIA_DATASET_NAME` if 
+necessary:
+     - [ ] file maps (may need to add or remove _Early_Release suffix!).
+     - [ ] column maps (if there are new or chaned columns).
+     - [ ] page maps (if the tab order changes).
+     - [ ] skip footer (if there is new, non-data content at the end of the file).
+     - [ ] skip rows (the early release data tends to have an extra, skippable row at 
+     the top of the file. Either add or subtract that row depending on whether you are
+     adding early release or final data).
+- [ ] Launch dagit and refresh the code location (run in your terminal 
+`dagster-webserver -m pudl.etl` and then open 
+`http://127.0.0.1:3000/locations/pudl.etl/jobs/etl_full` in a browser)
```

Comment:
is this link universal? I think the number bit at the front is specific to your computer (iirc). plus when you run the dagit script it tells you what link to launch so i wouldn't add that in there anyway in case it changes.

Reply:
good tip!! fixed.